packages/client/src/intros/shared/Intro.tsx

```diff
@@ -43,17 +45,19 @@ export default ({
 
   return (
     <>
-      <Paragraph gutterBottom={usableForBullets.length > 0 ? 8 : undefined}>
-        {t("introPage.common.check for permit intro")}{" "}
-        {usableForBullets.length > 0 && t("introPage.common.permit for")}
-      </Paragraph>
-
-      {usableForBullets.length > 0 && (
-        <List data-testid={INTRO_USABLE_FOR_BULLETS} variant="bullet">
-          {usableForBullets.map((bulletText) => (
-            <ListItem key={bulletText}>{bulletText}</ListItem>
-          ))}
-        </List>
+      <Paragraph>{introSentence}</Paragraph>
+      {usableForBullets?.length > 0 && (
+        <>
+          <Paragraph gutterBottom={8}>
+            {t("introPage.common.check for permit intro")}{" "}
```

Comment:
Why has this sentence been moved to the usableForBullets-case only? It's a generic sentence for all checkers.
The 'permit for' is specifically for checkers with usableForBullets.

Reply:
@svenjens changed this. I think it was because this checker is completely different and couldn't handle the `usableForBullets` anymore. Sven can you enhance?

Reply:
`It's a generic sentence for all checkers.`
This is not the case for the demolition checker. That starts different. This is the case for all checkers that have a usable For Bullets array.